andela/mrm_push
# CV3-74 add calendar to bouquet

## Description ##
Given a bouquet ID, this endpoint will allow calendars to be added to the bouquet if the shouldRefresh option is set to false. This means that instead of refreshing, calendars can be added manually to a bouquet.

## Type of change ##
Please select the relevant option

## How can This be tested ##
- Clone the repo: `git clone https://github.com/andela/mrm_push.git`
- Setup project according to `README.md`
- checkout to `story/CV3-74-add-calendars-to-bouquet` branch
- Activate the `virtualenv`
- run migration with the following commands
``` 
$ alembic upgrade head
```
- NB: make sure to update your `NOTIFICATION_URL` value in `.env` to reflect a google verified domain for notifications
 ## run the app
```
$ python manage.py runserver
```
- Make a `POST` request using postman on `http://127.0.0.1:5000/v2/channels`
- Request data:
```
{
    "calendar_id": "String", 
    "bouquet_id": Integer
}
```
## Running tests
```
$ pytest -v --cov
```
## Checklist: ##

## JIRA ##
[CV3-74](https://andela-apprenticeship.atlassian.net/browse/CV3-74?atlOrigin=eyJpIjoiNWEzZTVkNjY1ODJlNDUwYzgxNDgyNDI1MzZkZTEyODYiLCJwIjoiaiJ9)
